generate_qt_mappings.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+"""
+This script generates the Qt mapping file according to given Qt include
+directory
+
+Example usage :
+
+   $ ./generate_qt_mappings.py /usr/include/x86_64-linux-gnu/qt5 qt5_11.imp
+
+"""
+
+from __future__ import print_function
+import argparse
+import glob
+import os
+import re
+import sys
+
+
+def main(qt_include_dir, output_file):
+    symbols_map = []
+    includes_map = []
+
+    headers = glob.glob(os.path.join(args.qt_include_dir, '**/*[!.h]'))
+    for header in headers:
+        if os.path.isdir(header):
+            continue
+
+        class_name = os.path.basename(header)
+        module_name = os.path.basename(os.path.dirname(header))
+
+        symbols_map += ['{{ symbol: [ "{}", "private", '.format(class_name)
```

Comment:
I realize it's old-school and goes against current recommendations, but I much prefer the C-style format strings to `.format`, because it's more succinct and I more readable without all the curly braces.